frontend/src/styles/Courses.css

```diff
@@ -128,23 +128,12 @@
   z-index: 1;
 }
 
-.course-card:hover:active, .course-card-shadow {
-  cursor: grabbing;
-  box-shadow: 0 14px 28px rgba(0,0,0,0.25), 0 10px 10px rgba(0,0,0,0.22);
-  z-index: 1;
-}
-
-/* don't show react-beautiful-dnd's placeholder in search results */
-.placeholder-course + div {
```

Comment:
react-beautiful-dnd has changed in the updated version, so this placeholder course hack doesn't work anymore. We do it in a different way in this PR.